### ISSUE TITLE
hi, malihu-custom-scrollbar has preformance trick.

### ISSUE THREAD

**Issue by hetz** [malihu/malihu-custom-scrollbar-plugin#262]
when I setting  advanced-> updateOnImageLoad = false,  scrollbar preformance improved a lot.

**Comment by malihu** [comment]
Yes indeed. All auto-update options when disabled increase performance. On the next version I'll change at least 2 of them to perform much better. Thanks for the feedback!

**Comment by shlima** [comment]
We are waiting updates. The best scroll plugin for my choice.  :relaxed:

**Comment by adyz** [comment]
Any updates? Is this still true? 

Please check this video, looks like your plugin abuses the mouse-wheel events: https://www.youtube.com/watch?v=6-D_3yx_KVI
